scripts/setup/environment.json

```diff
@@ -1,7 +1,6 @@
 {
     "cipd_package_files": [
         "third_party/pigweed/repo/pw_env_setup/py/pw_env_setup/cipd_setup/arm.json",
```

Comment:
TODO, I need to remove scripts/setup/pigweed.json file. I avoided it to prevent conflict with https://github.com/project-chip/connectedhomeip/pull/29952 which will likely be merged before the PR is completed